BuildingSync.xsd

```diff
@@ -3382,6 +3402,30 @@
         </xs:complexType>
       </xs:element>
       <xs:element ref="auc:UserDefinedFields" minOccurs="0"/>
+      <xs:element minOccurs="0" name="AnnualAverageCarbonEmissions">
+        <xs:annotation>
+          <xs:documentation>Annual Average Carbon Emissions. (kg CO2e)</xs:documentation>
```

Comment:
This is really a GHG metric since it is CO2e which is each greenhouse gases multipled by global warming potential. So Nitrus Oxide has no carbon, per se, but has a GWP of ~300, so it is more "potent" than 1 kg CO2. Don't make changes yet on the term `AnnualAverageCarbonEmissions`, let's see what others think.

Reply:
Maybe we just name it AnnualAverageGHGEmissions to cover all scenarios?  However, if we do this then should we change all (e.g. AnnualSavingsAverageCarbonEmissions to AnnualSavingsAverageGHGEmissions?)

Reply:
that's how it is here: https://github.com/BuildingSync/schema/blob/ghg-example/BuildingSync.xsd#L3108.  We could keep that pattern... Let me know if you want that change or if you want to wait.

Reply:
@kflemin let's change it from carbon to GHG throughout.  It's more accurate so it makes sense to just include all GHG emissions in addition to carbon, even if they are small.

Reply:
@laurenliz22, done!